src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In Javascript, combination of function and its lexical environment is called?",
```

Comment:
Please make sure to use proper capitalization for JavaScript. Also let's go with this rewording. 

```suggestion
      "In JavaScript, what is it called when you have a combination of a function and its lexical environment?",
```